src/tx/index.js

```diff
@@ -97,13 +97,15 @@ async function loadMetaData(address, base, timeout = 2000) {
   const res = await getTx(path, base, timeout);
   accNum = _.get(res, 'data.result.value.account_number');
   seqNum = _.get(res, 'data.result.value.sequence');
-  if (!(accNum || seqNum)) {
-    throw new Error(
-      'account number or sequence number from rest server are undefined'
-    );
+  let signMetaData
+  if ((typeof accNum === 'undefined' || typeof seqNum === 'undefined')) {
+    signMetaData = {
+      account_number: '0',
+      sequence: '0',
+    };
```

Comment:
If only 1 of the variables is undefined both will be reset to 0 as it's assuming if one is undefined, both will be undefined. While potentially a valid assumption, should change `||` to `&&` on line 101 as it reflects the contents of this if statement. Or update the body with `account_number: typeof accNum === 'undefined' ? '0' : accNum`.